distributions/openhab-runtime-karaf/src/main/resources/runtime/etc/jetty.xml

```diff
@@ -0,0 +1,244 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_0.dtd">
+
+
+<!-- =============================================================== -->
+<!-- Configure the Jetty Server -->
+<!-- -->
+<!-- Documentation of this file format can be found at: -->
+<!-- http://wiki.eclipse.org/Jetty/Reference/jetty.xml_syntax -->
+<!-- =============================================================== -->
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+
+	<!-- =========================================================== -->
+	<!-- Set handler Collection Structure -->
+	<!-- =========================================================== -->
+
+	<Get name="handler">
+		<Call name="addHandler">
+			<Arg>
+				<New id="Rewrite" class="org.eclipse.jetty.rewrite.handler.RewriteHandler">
+					<!-- Add rule in order to take care of the X-Forwarded-Scheme header -->
+					<Call name="addRule">
+						<Arg>
+							<New class="org.eclipse.jetty.rewrite.handler.ForwardedSchemeHeaderRule">
+								<Set name="header">X-Forwarded-Scheme</Set>
+								<Set name="headerValue">https</Set> <!-- if this is unset, any value will match against the rule -->
+								<Set name="scheme">https</Set>
+							</New>
+						</Arg>
+					</Call>
+					<Call name="addRule">
+						<Arg>
+							<New class="org.eclipse.jetty.rewrite.handler.ForwardedSchemeHeaderRule">
+								<Set name="header">X-Forwarded-Scheme</Set>
+								<Set name="headerValue">http</Set> <!-- if this is unset, any value will match against the rule -->
+								<Set name="scheme">http</Set>
+							</New>
+						</Arg>
+					</Call>
+
+					<!-- show the dashboard as default -->
+					<Call name="addRule">
+						<Arg>
+							<New class="org.eclipse.jetty.rewrite.handler.RedirectRegexRule">
+								<Set name="regex">/$</Set>
+								<Set name="replacement">/start/index</Set>
+							</New>
+						</Arg>
+					</Call>
+				</New>
+			</Arg>
+		</Call>
+	</Get>
+	<Get name="handler">
+		<Call name="addHandler">
+			<Arg>
+				<New class="org.eclipse.jetty.server.handler.ContextHandler">
+					<Set name="contextPath">/webapps</Set>
```

Comment:
As we do not allow real "webapps" here, we might also want to go for a different name. I don't really have a good suggestion. On the file system, we could maybe simply call it `conf/html`, while on the url it could be `/static`.